docs/pages/reference/cli/tbot.mdx

```diff
@@ -85,6 +86,7 @@ configuring either file or POSIX ACL permissions.
 | `--reader-user`     | Enables POSIX ACLs and defines the Linux user that will read short-lived certificates from `--destination-dir`.    |
 | `--init-dir`        | If using a config file and multiple destinations are configured, controls which destination dir to configure.      |
 | `--clean`           | If set, remove unexpected files and directories from the destination.                                              |
+| `--log-format`      | Controls the format of output logs. Can be `json` or `text`. Defaults to `text`.                                   |
```

Comment:
@ptgott I think there was a project to auto-generate the CLI reference at some point. It would make it easier to maintain this part of the docs while potentially lowering the amount of the docs-review workload. What would be needed to make it a reality?